data_loader.py

```diff
@@ -73,5 +84,15 @@ def _is_active_employee(employee, zenefits_session):
     return False
 
 
+def _post_to_builder(builder_session, url, data, files=None):
```

Comment:
I like the direction this is going. Since this is always expecting a builder session, do you think it would make sense to make a super-light builder client class? Something like:
```python
class BuilderClient(object):
    """
    Helper class for making requests to the Builder API which handles things like raising exceptions on invalid 
    responses etc.
    """
    def __init__(self):
        self.builder_session = requests.Session()
        self.builder_session.headers.update({"Authorization": f"JWT {settings.builder_api_key}"})
    def post(...):
        ...
    def get(...):
        ....
```

Reply:
I like this! added it here https://github.com/Guidebook/guidebook-zenefits-integration/pull/2/commits/2cb7052176eca46bdf6a2ee5100339f81fa3f73c